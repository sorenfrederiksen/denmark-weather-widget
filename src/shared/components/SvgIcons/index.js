// @flow

import React from 'react';

//
// Component
// --------------------------------------------------------------------

const SvgIcons = () => (
  <svg xmlns="http://www.w3.org/2000/svg" style={{ position: 'absolute', width: 0, height: 0 }}>
    <symbol id="icon-tv2-logo" viewBox="0 0 31.67 14.17">
      <path
        fill="currentColor"
        fillRule="evenodd"
        d="M28.71 1h-9.24a1 1 0 0 0-.8.43l-2 3H28a.86.86 0 0 1 .94.76A1.3 1.3 0 0 1 27.6 6.4h-5a1 1 0 0 0-.8.43l-2 2.94H27c2.82 0 5.76-2.45 5.65-5.16a3.87 3.87 0 0 0-4-3.61M13.64 4.38H1l2-2.95A1.08 1.08 0 0 1 3.79 1h12.64l-2 3a1 1 0 0 1-.79.43"
        transform="translate(-1 -1)"
      />
      <path
        fill="currentColor"
        fillRule="evenodd"
        d="M6.32 14.1l.81-7.7h3.75l-.81 7.77L15 6.83a1.08 1.08 0 0 1 .8-.43h3.63l-5.63 8.35a1.06 1.06 0 0 1-.8.42H7.29a1 1 0 0 1-1-1.07M28 15.17H16.12l2-2.95a1.1 1.1 0 0 1 .8-.42h11.85l-2 2.95a1 1 0 0 1-.79.42"
        transform="translate(-1 -1)"
      />
    </symbol>
    <symbol viewBox="0 0 32 32" id="icon-search">
      <path
        d="M21.08 18.573l10.4 10.4a1.773 1.773 0 1 1-2.508 2.507l-10.4-10.4a11.62 11.62 0 0 1-6.902 2.26C5.225 23.34 0 18.115 0 11.67S5.225 0 11.67 0s11.67 5.225 11.67 11.67c0 2.583-.84 4.97-2.26 6.903zm-1.287-6.903a8.123 8.123 0 1 0-16.247 0 8.123 8.123 0 0 0 16.247 0z"
        fill="currentColor"
        fillRule="evenodd"
      />
    </symbol>
  </svg>
);

export default SvgIcons;
