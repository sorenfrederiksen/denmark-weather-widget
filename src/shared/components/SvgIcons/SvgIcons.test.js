import React from 'react';
import { shallow } from 'enzyme';
import SvgIcons from './index';

describe('(Component) SvgIcons', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SvgIcons />);
  });

  it('should match snapshot.', () => {
    expect(wrapper).toMatchSnapshot();
  });
  /*
  it('should appear deactivated, when deactivated.', () => {
    const newProps = {
      ...props,
      isLive: false,
    };

    wrapper = shallow(<Features {...newProps} />);
  });*/
});
