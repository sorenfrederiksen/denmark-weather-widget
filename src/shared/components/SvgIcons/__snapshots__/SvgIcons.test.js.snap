// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) SvgIcons should match snapshot. 1`] = `ShallowWrapper {}`;
